beacon-chain/main.go

```diff
@@ -155,6 +157,21 @@ func main() {
 			}
 		}
 
+		// Expand the path if web3 endpoint is a file
```

Comment:
Can we refactor this into its own helper function?

Reply:
ok